packages/components/radio/src/Radio.tsx

```diff
@@ -17,7 +17,7 @@ export const Radio = forwardRef<HTMLButtonElement, RadioProps>(
     const disabled = disabledProp || context.disabled
 
     return (
-      <RadioLabel disabled={disabled}>
+      <RadioLabel className={className} disabled={disabled}>
```

Comment:
@andresz1 Is this ok to pass `className` in label, not input?

Reply:
Yes! I think that is ok